src/pages/proposal-commons/proposal-details/proposal-details.module.scss

```diff
@@ -1,87 +1,115 @@
 @import '../../../styles/variables.module.scss';
+@import '../../../styles/fonts.module.scss';
 
 .backBtn {
   button {
```

Comment:
Earlier today I pushed the PR with the Modals, which removes the `Button` component wrapper. Please lose this `button` after you incorporate the new changes.

Reply:
Thanks for thinking about this 🙏 I have rebased this PR and removed the wrapper.